backend/src/test/java/de/neuefische/backend/controller/UserControllerTest.java

```diff
@@ -53,9 +54,7 @@ void addUser_expect_valid_user_and_an_id() throws Exception {
 
         User actualResult = objectMapper.readValue(result.getResponse().getContentAsString(), User.class);
 
-        assertInstanceOf(User.class, actualResult);
         assertFalse(actualResult.id().isEmpty());
-        assertFalse(actualResult.id().isBlank());
```

Comment:
You should check if the name and visitedCountries got save like you expected it.